versions.yml

```diff
@@ -24,41 +24,41 @@ modGroups:
       homepage: https://wynntils.com
       modrinthId: dU5Gb9Ab
       desc: A Minecraft mod that adds a lot of quality of life features to Wynncraft
-      url: https://athena.wynntils.com/version/download/artemis/release/fabric?1.20.2
+      url: https://athena.wynntils.com/version/download/artemis/beta/fabric?1.21
```

Comment:
Can we use modrinth here..? That would make it so Wynntils is not updated by launchy.

Reply:
Well we would have to update it each time we update wynntils, but like you know, we either need to put here more work or it won't be able to download at all. Well, the best way would be to fix Athena itself so it works